Highway.h

```diff
@@ -2,15 +2,20 @@
 
 #include <vector>
 #include <algorithm>
+#include "Vehicle.h"
+
 
 struct Highway
 {
+public:
     void changeSpeed(int newSpeed);
     void addVehicle(Vehicle* v);
     void removeVehicle(Vehicle* v);
+    int speedLimit = 65;
+    std::vector<Vehicle*> vehicles;
```

Comment:
These were private, make them private again. 

- [ ] Re-read the instructions for what you're supposed to do to get the Highway and HighwayPatrol to work together and revise.